flow/designs/asap7/gcd/config.mk

```diff
@@ -4,6 +4,7 @@ export DESIGN_NAME            = gcd
 
 export VERILOG_FILES          = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
 export SDC_FILE               = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
+export ABC_CLOCK_PERIOD_IN_PS = 340
```

Comment:
@ravi-varadarajan  I see some designs (including this) have a mismatch between `ABC_CLOCK_PERIOD_IN_PS` and the value from `set clk_period` is this intentional?

Reply:
It need not be an exact match. But it probably good to keep it in sync.  There is also another issue where if this variable is not set, synth.tcl does not use any abc_script file. I was going to fix that as part of this PR too, but decided to fix that as a dedicated PR.

Reply:
If there is no benefit of having different values, it would be nice that one depends on the other.
Either use the env variable in the SDC if this is possible or populate the env variable with a grep regex or similar from the SDC file.

Reply:
In most platforms we have a default of:

export ABC_CLOCK_PERIOD_IN_PS ?= $(shell grep -E -o -m 1 "\-period\s+\S+" $(SDC_FILE) | awk '{print $$2}')

so there is no need to set it unless you have to for some particular reason (like multiple clocks in bp_single.  Notably intel22 is missing this default (I think it should be added).

Reply:
Though doing this will break that default:

set clk_name clk
set clk_period 2600
create_clock -name $clk_name -period $clk_period [get_ports clk]

since you'll just get $clk_period instead of the value.  This seems like a lot of duplication.

Reply:
I think we can take this up in a separate PR